src/utils/validation/schema.js

```diff
@@ -124,6 +124,22 @@ const normalBolus = {
   wizard: { type: 'string', pattern: patterns.id, ...optional },
 };
 
+const automatedBolus = {
```

Comment:
Looks like the only difference between this and the `normalBolus` schema is the `subType`.  Would it make more sense to simply add `automated` to the `subType` enum for `normalBolus`?

If we can do that, it will also reduce (or rather, not increase) the number of schemas we need to compare against for bolus datums.

Reply:
Agreed, that makes it much simpler.